src/main/groovy/com/brightsparklabs/gradle/docs/DocsPlugin.groovy

```diff
@@ -467,14 +473,20 @@ public class DocsPlugin implements Plugin<Project> {
                      * be overridden in Asciidoc documents. See:
                      * - https://docs.asciidoctor.org/asciidoc/latest/attributes/assignment-precedence/
                      */
-                    attributes \
-                            'chapter-label@'      : '',
-                            'icons@'              : 'font',
-                            'imagesdir@'          : project.file(config.buildImagesDir),
-                            'numbered@'           : '',
-                            'source-highlighter@' : 'coderay',
-                            'toc@'                : config.tocPosition,
-                            'title-logo-image@'   : config.titleLogoImage
+
+                    Map<String,Object> pluginAttributes = [
+                        "chapter-label@"        : '',
```

Comment:
Likewise, `DEFAULT_ASCIIDOC_ATTRIBUTES` constant please.

NOTE: ASCIIDOC not ASCIIDOCTOR as the attributes are asciidoc concepts, irrelevant of the tool

Reply:
We can move the attributes map into a constant. We will need to replace the current references to config with their default values as outlined in config instead. I will also need to find a different method of setting the image-dir as it is currently reliant on being able to use 'project' to properly define it's location relative to the project root.